samples/snippets/publisher.py

```diff
@@ -103,6 +103,74 @@ def create_topic_with_kinesis_ingestion(
     # [END pubsub_create_topic_with_kinesis_ingestion]
 
 
+def create_topic_with_cloud_storage_ingestion(
+    project_id: str,
+    topic_id: str,
+    bucket: str,
+    input_format: str,
+    text_delimiter: str,
+    match_glob: str,
+    minimum_object_create_time: str
+) -> None:
+    """Create a new Pub/Sub topic with Cloud Storage Ingestion Settings."""
+    # [START pubsub_create_topic_with_cloud_storage_ingestion]
+    from google.cloud import pubsub_v1
+    from google.protobuf import timestamp_pb2
+    from google.pubsub_v1.types import Topic
+    from google.pubsub_v1.types import IngestionDataSourceSettings
+
+    # TODO(developer)
+    # project_id = "your-project-id"
+    # topic_id = "your-topic-id"
+    # bucket = "your-bucket"
+    # input_format = "text"
```

Comment:
Is text the only input_format that is supported, or should we make this explicit to the user that they could set the input_format based on their needs.

Reply:
I've added a comment for the possible input_format types.